src/databricks/azext_databricks/_help.py

```diff
@@ -34,6 +34,16 @@
       - name: Clean the workspace's tags.
         text: |-
                az databricks workspace update --resource-group MyResourceGroup --name MyWorkspace --tags ""
+      - name: Assign identity for storage account.
+        text: |-
+               az databricks workspace update --resource-group MyResourceGroup --name MyWorkspace --assign-identity

```

Comment:
does it support configuring identity type?

Reply:
just confirm, spec doesn't mention identity be configured at creation time

Reply:
identity type cannot be configured, it's managed by databricks service. It can only be enabled in update command in the spec.